src/map/script.cpp

```diff
@@ -13248,7 +13248,14 @@ BUILDIN_FUNC(getwaitingroomstate)
 
 	if( nd == nullptr || (cd=(chat_data *)map_id2bl(nd->chat_id)) == nullptr )
 	{
-		script_pushint(st, -1);
+		switch(type){
+			case 4:
+			case 5:
+			case 16:
+				script_pushconststr(st,"null");
+				break;
+			default: script_pushint(st,-1);break;
```

Comment:
Could we list the "known" types and (at least) give a warning when an unknown value is used?
I mean I would prefer to return SCRIPT_CMD_FAILURE in this case.
It is okay to return -1 for known types.